dashboard/app/models/user.rb

```diff
@@ -107,6 +107,14 @@ class User < ActiveRecord::Base
     closed_dialog
     nonsense
   ).freeze
+
+  # Notes:
+  #   data_transfer_agreement_source: Indicates the source of the data transfer
+  #     agreement.
+  #   data_transfer_agreement_kind: "0", "1", etc.  Indicates which version
+  #     of the data transfer agreement string the user to agreed to, for a given
+  #     data_transfer_agreement_source.  This value should be bumped each time
+  #     the corresponding user-facing string is updated.
```

Comment:
If it's a version, why not call it `data_transfer_agreement_version`?

Reply:
Great question.  This should parallel what we do with `EmailPreference` and @sureshc gets to own the schema on that one.  We didn't get super crisp on whether that one is strictly a version or more of a tag, even though we've started with what look like version numbers.